panel/lxqtpanel.cpp

```diff
@@ -640,7 +635,8 @@ void LxQtPanel::showAddPluginDialog()
 void LxQtPanel::addPlugin(const LxQt::PluginInfo &desktopFile)
 {
     QString settingsGroup = findNewPluginSettingsGroup(desktopFile.id());
-    loadPlugin(desktopFile, settingsGroup);
+    if (0 != loadPlugin(desktopFile, settingsGroup))
```

Comment:
Oh, sorry, I thought it was `|=`, not `!=`. That's what very few hours of sleep get you.